charts/mlp/templates/deployment.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "mlp.fullname" .}}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "mlp.name" .}}
```

Comment:
we should use a helper template to render equivalent set of labels for all resources

Reply:
can you elaborate more here?  Do you mean to use something like [this](https://github.com/caraml-dev/helm-charts/pull/6/files#diff-747d8fda350f0d7f200316e02659d4441c29d2087085dabeb660b464c97066beR32) from the other PR? 

or to create a library chart with helper functions to be used in all the sub-charts?

Reply:
ok, I think I got it. You mean to add a common helper template for labels. I misunderstood it in the last reply above. I will make the necessary changes.

Reply:
Added.